configuration/rules-dsl.md

```diff
@@ -206,6 +206,40 @@ For example, one z-wave device can be "zwave:device:c5155aa4:node14".
 > Note: You need to use quotes around `thingUID` if it contains special characters such as ':'.
 > The parser will not recognize the whole text correctly without quotes.
 
+{: #channel-based-triggers}
+### Channel-based Triggers
+
+You can listen for triggers from specific *trigger channels* that are provided by some addons.
+You can decide whether you want to catch only a specific trigger or any.
+Here is the syntax for all these cases (parts in square brackets are optional):
```

Comment:
All what cases?  I guess if you answer the comment above, then you will also make clear what cases you are talking about.

Reply:
This is the intro for the super duper master example, which covers all possible cases.

For the sake of simplicity i will remove the all.
We only got 2 cases, so "all" is not really needed here.